src/pages/Address/AddressAssetComp.tsx

```diff
@@ -159,7 +159,7 @@ export const AddressNRC721Comp = ({ account, isRGBPP }: { account: NRC721; isRGB
     <AddressAssetComp
       isRGBPP={isRGBPP ?? false}
       href={`/nft-collections/${collection?.typeHash}`}
-      property={!symbol ? '?' : `#${amount}`}
+      property={!symbol ? '?' : `#${sliceNftName(amount, 20)}`}
```

Comment:
How about using `text-overflow: ellipsis` by css

Reply:
Any update on this conversation?

Reply:
> Any update on this conversation?

Forgot to push to GitHub, pushed just now.

Reply:
Is `sliceNftName` still necessary after using the `text-overflow: ellipsis`?

Reply:
> Is `sliceNftName` still necessary after using the `text-overflow: ellipsis`?

have changed in https://github.com/Magickbase/ckb-explorer-frontend/pull/289/commits/6e44be7d134a6c222acb2022bcc24b0fda8e0483